fixtures/cassettes/orders_responses.yml

```diff
@@ -92,6 +92,37 @@ d0c2a1b2a0fd3078fcb7492bc2ea067cd82834e8:
       encoding: UTF-8
       string: "<errors><error>Could not find order with guid [B83E748F-BEEF-528A-0265-CA6DDA7A8A78].</error></errors>"
     http_version: 
+4c6e7a050df716108e7baa1c711249ecc3aa08e9:
+  response:
+    status:
+      code: 404
+      message: 
+    headers:
+      server:
+      - Apache-Coyote/1.1
+      cache-control:
+      - no-cache
+      x-ua-compatible:
+      - IE=Edge,chrome=1
+      set-cookie:
+      - _EchoRestRails_session=BAh7BiIPc2Vzc2lvbl9pZCIlNjk3OGMwMmY3M2MyZjRlZTRiYjIyNzcyOGZhZjQzNDg%3D--8ba4d2eab87c4f5774e2732c87be8be2911b2dae;
+        path=/; HttpOnly
+      x-runtime:
+      - '0.420000'
+      content-type:
+      - application/xml;charset=utf-8
+      vary:
+      - Accept-Encoding
+      date:
+      - Mon, 28 Sep 2015 17:28:11 GMT
+      connection:
+      - close
+      transfer-encoding:
+      - chunked
+    body:
+      encoding: UTF-8
+      string: "<errors><error>Could not find order with guid [B83E748F-BEEF-528A-0265-CA6DDA7A8A78].</error></errors>"
```

Comment:
Is this supposed to be an error? It looks like you may be submitting an order when you shouldn't. The only actual orders should be in hand-edited.